include/oneapi/dpl/internal/common_config.h

```diff
@@ -31,7 +31,21 @@
 // - New TBB version with incompatible APIs is found (libstdc++ v9/v10)
 #    if __has_include(<tbb/version.h>)
 #        ifndef PSTL_USE_PARALLEL_POLICIES
+//           _GLIBCXX_RELEASE can be undefined if oneDPL is included before any STL headers
+//           It is not an issue if PSTL_USE_PARALLEL_POLICIES would be used after _GLIBCXX_RELEASE definition
 #            define PSTL_USE_PARALLEL_POLICIES (_GLIBCXX_RELEASE != 9)
```

Comment:
So, public macro definition should be removed?
Why it was no enough so far?

Reply:
No, It is still required to be defined for the case when the user only includes `<oneapi/dpl/execution>`. 
In that case `__PSTL_USE_PAR_POLICIES` and `__PSTL_PAR_BACKEND_TBB` while including `<execution>` would be defined according to the public macro value.